.github/CODEOWNERS

```diff
@@ -4,9 +4,10 @@
 # The last matching pattern takes precedence.
 # https://help.github.com/articles/about-codeowners/
 
-*                               @spotify/backstage-core
-/docs/features/techdocs         @spotify/techdocs-core
-/plugins/techdocs               @spotify/techdocs-core
-/plugins/techdocs-backend       @spotify/techdocs-core
-/packages/techdocs-cli          @spotify/techdocs-core
-/packages/techdocs-container    @spotify/techdocs-core
+*                            @spotify/backstage-core
+/docs/features/techdocs      @spotify/techdocs-core
+/packages/techdocs-cli       @spotify/techdocs-core
+/packages/techdocs-container @spotify/techdocs-core
+/plugins/tech-stack          @easylo
```

Comment:
@stefanalund @benjdlambert Is it intended to deeply integrate 3rd party plugins like this into the repository?

Concerns:
* uncontrolled split of codeowners/ownership
* fixed dependency integration of all plugins (see `package.json`)
* update mechanism for 3rd party/contributed plugins

Reply:
Yeah I'd request that we don't add this entry to `CODEOWNERS` for now. We're still not quite sure what the long-term setup of the plugin echosystem will be like, except that we definitely want to support plugin development outside of this repo.

It's recommended to keep plugins in this repo for now to ensure that they get updated for breaking changes, but I don't think we know yet if that's going to be a good long-term model or not.

Reply:
ok, removed